packages/cli/test/utils/validator.ts

```diff
@@ -73,8 +72,6 @@ export async function startValidatorWithKeyManager(
     await gracefullyStopChildProcess(validatorProc, 3000);
   };
 
-  afterEach(stopValidator);
```

Comment:
In this version of vitest, behavior of hooks is changed. It's more stable to have all hooks listed in the test file to make sure the order of hooks works as expected.